utility/utility.py

```diff
@@ -138,3 +139,10 @@ def get_property(request_iterator):
 
         d["stream"] = stream
     return d
+
+
+def is_expired(timestamp):
+    """Determine if a given timestamp has expired"""
+    if timestamp <= 0 or time.time() >= timestamp:
+        return True
+    return False
```

Comment:
Please include a unit test. Look at the `test_utility.py`.

Make sure you test for
- 0
- negative numbers
- expired
- non expired
- and `None`